src/Data/Unrestricted/Linear/Internal/Dupable.hs

```diff
@@ -1,28 +1,50 @@
+{- options_ghc -ddump-simpl -dsuppress-type-applications -dsuppress-coercions #-}
 {-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE EmptyCase #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE LinearTypes #-}
 {-# LANGUAGE PolyKinds #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
 {-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# OPTIONS_HADDOCK hide #-}
 
 module Data.Unrestricted.Linear.Internal.Dupable
   ( Dupable (..),
+    genericdupR,
```

Comment:
I think that we should change it to `genericDupR` for consistency

Reply:
In other packages, I think the lower case convention tends to be used, but I guess we have `genericTraverse` now? I'm not sure what the right path is.

Reply:
We have `genericTraverse`, `genericPure`, `genericLiftA2`... so I think we should use `genericDupR`. The question is, what should we do with `gXXXX` functions (but I think this is less important because they're not publicly exposed)?

Reply:
Doesn't matter. There's no point in exposing them that I know of.